packages/shared/components/inputs/AssetAmountInput.svelte

```diff
@@ -40,6 +40,7 @@
     $: marketAmount = getMarketAmountFromAssetValue(bigAmount, asset)
     $: max = parseCurrency(formatTokenAmountDefault(availableBalance, asset?.metadata, unit, false))
     $: rawAmount = bigAmount?.toString()
+    $: asset = $visibleSelectedAccountAssets?.[$activeProfile?.network?.id]?.baseCoin ?? asset
```

Comment:
I havent tested the code but from reading the changes I think this can lead to issues if for some reason the `visibleSelectedAccountAssets` changes, `asset` is binded in `<AssetDropdown bind:asset readonly={disableAssetSelection} />` and this code looks like its going to override it

Reply:
Indeed, this code doesnt work, you cant even select a native token